ptx/sec_deriv_prodquot.ptx

```diff
@@ -440,25 +441,16 @@
   <theorem xml:id="thm_deriv_trig">
     <title>Derivatives of Trigonometric Functions</title>
     <statement>
-      <p>
-        <ol cols="2">
-          <li>
-            <p>
-              <m>\displaystyle \lzoo{x}{\sin(x)} = \cos(x)</m>
 
-      <idx><h>derivative</h><h>trigonometric functions</h></idx>
-
-            </p>
-          </li>
+          <idx><h>derivative</h><h>trigonometric functions</h></idx>
```

Comment:
Can `<idx>` go here? I thought it had to be inside a `<p>`.

Reply:
You're right that it can't go here. Schema validation confirms that. Formerly it was inside one of the list items, which seems logically incorrect given that it is the marker for (all) trig derivatives. I moved it to follow the theorem's title, and that passes validation. I will commit and force push after reviewing the other comments.